internal/ingress/controller/store/store_test.go

```diff
@@ -193,30 +184,34 @@ func TestStore(t *testing.T) {
 					},
 				},
 			},
-		}, clientSet)
+		}, clientSet, t)
+
+		err := framework.WaitForIngressInNamespace(clientSet, ns, ing.Name)
 		if err != nil {
-			t.Errorf("unexpected error creating ingress: %v", err)
+			t.Errorf("unexpected error waiting for secret: %v", err)
```

Comment:
(my favorite review comment, sorry for repeating myself) Log messages should start with a capital letter to differentiate them from error messages returned by functions.
Besides errors are never expected, so removing the word "unexpected" from all these log messages makes messages shorter and to the point. But that's obviously a nit pick, these messages are not user facing so feel free to skip that comment.

Reply:
Done